tests/sqllogictests/suites/base/03_common/03_0048_nondeterministic_update.test

```diff
@@ -0,0 +1,62 @@
+statement ok
+set error_on_nondeterministic_update = 0;
```

Comment:
For the test we need two parts:
For the same dataset and same operations, check the differences.

1. error_on_nondeterministic_update is false
2. error_on_nondeterministic_update is true